### PR TITLE
Couple of FAQ updates.

### PR DESCRIPTION
**Pull request by sudswasavi**

### CODE REVIEW

**Review by monotosh-avi — APPROVED** [review]

File: docs/AKO/faq.md [hunk 1]
```diff
@@ -184,10 +188,26 @@ No. Users can only use service of type ClusterIP as backend of Ingresses in this
...
 
+#### What are the steps for ServiceType change?
+
+The `serviceType` in AKO can be changed from `ClusterIP` to `NodePortLocal` or `NodePort`. The `serviceType` change is considered disruptive.
+Hence before the `serviceType` change, all the existing AKO configuration must be deleted. This can be achieved as follows:
+
+  - Set the `deleteConfig` flag to `true`.
+  - Wait for AKO to delete all the relevant Avi configuration and update the deletion status in AKO's statefulset status.
+  - Set the `deleteConfig` flag to `false`
+  - Change the `serviceType`
```
Comment (rathinikunj) [inline docs/AKO/faq.md:199]: change the serviceType should be done before setting the deleteConfig to False, else AKO will again start syncing based on the old serviceType

**Review by rathinikunj — CHANGES_REQUESTED** [review]
Hi Sudipto,

Requesting just one change,

File: docs/AKO/faq.md (hunk 1 above)
Comment (sudswasavi) [reply docs/AKO/faq.md:199]: Thanks @rathinikunj . Changing it.

Comment (sudswasavi) [reply docs/AKO/faq.md:198]: Done.

**Review by rathinikunj — APPROVED** [review]
Looks good now

**Pull request merged**